docs/topics/mta-requirements-for-java-analyzer.adoc

```diff
@@ -6,11 +6,11 @@
 [id="mta-requirements-for-java-analyzer_{context}"]
 = Adding minimum requirements for Java analyzer and discovery
 
-There is a minimum requirement for the Java analyzer, and also the discovery task, which by default is set to 2 GB.
+The default minimum requirement for the Java analyzer, and also the discovery task, is set to 2.5 GB.
 
 While this minimum requirement can be lowered to 1.5 GB, it is not recommended.
```

Comment:
I think this line should be removed since it may cause analysis failure.
But I'll defer to @Jeff Ortel on this.